aptos-move/framework/aptos-framework/sources/jwks.move

```diff
@@ -25,12 +27,18 @@ module aptos_framework::jwks {
     friend aptos_framework::genesis;
     friend aptos_framework::reconfiguration_with_dkg;
 
+    /// We limit the size of a `PatchedJWKs` resource installed by a dapp owner for federated keyless accounts.
+    /// TODO: I am not sure how large we want to allow this to be. If too large => validators waste work reading it for invalid TXN signatures.
+    const MAX_FEDERATED_JWKS_SIZE_BYTES: u64 = 10 * 1024; // 10 KiB
```

Comment:
do we want to do some performance analysis of this?
we could fall back to our existing threat model that says we anticipate honest but curious and not malicious VFN -> V interactions and assume we'd only see legitimate asks here...

Reply:
Eventually, but now now if we plan to merge in 1.19

Reply:
Actually, @heliuchuan, do we know how big we expect the JWKs for providers like Auth0, AWS Cognito to be?

Reply:
How conservative can we be here, because it may make more sense to start conservatively.

Reply:
Maybe we should confirm these numbers with @msmouse too.

Reply:
@msmouse says 1-2KiB ideally. No more than 4KiB.

Reply:
Setting it to 2KiB for now.